docs/custom_elements.qmd

```diff
@@ -0,0 +1,194 @@
+# Creating and deploying custom elements
```

Comment:
I find the term "custom elements" a bit unclear. If I was searching for how to do this in Vetiver, I think I would search for something like:

- How to deploy custom models in Vetiver.
- How to deploy unsupported models in Vetiver.
- How to include custom code in Vetiver.
- How to deploy a Vetiver model with a custom pipeline.

Are you committed already to the term "custom elements"? I also notice we use a few different terms in this doc:

- custom element
- custom model
- custom handler

Reply:
"Custom elements" was me trying to find single word that worked for unsupported models/custom code/custom models. I'm open to suggestions if you have something else in mind! Would just calling it custom code make more sense? Even though it's sometimes not custom _code_, but rather an unsupported model?

If I aligned on one word (custom element or otherwise), and then used 

- `CustomHandler` rather than "custom handler" 
- "unsupported model" rather than "custom model"

would that be a better indication of what I am referring to?